phoebe/backend/contacts_smoothing.py

```diff
@@ -0,0 +1,241 @@
+import numpy as np
+import matplotlib.pyplot as plt
+from itertools import combinations
+
+
+def _isolate_neck(coords_all, teffs_all, cutoff = 0.,component=1, plot=False):
```

Comment:
can you please document this and other functions with a docstring?

Reply:
on it!